### ISSUE TITLE
prefs namespace

### ISSUE THREAD

**Issue by bwbohl** [Edirom/Edirom-Online#144]
@roewenstrunk what about introducing a namespace for edirom pref files, e.g.:
```http://www.edirom.de/ns/prefs/1.0```

**Comment by bwbohl** [comment]
I ask this because I started developing a schema for prefs (https://github.com/Edirom/edirom-edition-schema/blob/develop/prefs/v1.0/prefs.rng) and would like to automatically assign it for validation in a prefs oXygen framework developed at https://github.com/Edirom/edirom-oXygen-framework

**Comment by peterstadler** [comment]
I don't think that this has to break backwards compatibility! You could simply modify the functions to support both flavors of pref files.

**Comment by bwbohl** [comment]
right

**Comment by daniel-jettka** [comment]
> I don't think that this has to break backwards compatibility! You could simply modify the functions to support both flavors of pref files.

How would you do that? Like... 
(a) `($preferences//pref:entry, $preferences//*:entry)[1]`
(b) `$preferences//*:entry`

or completely different?

**Comment by peterstadler** [comment]
Yes, exactly!
I think option a) would be more precise and facilitates a deprecation and removal at some time in the future (by simply removing one item).
it should probably read `($preferences//pref:entry, $preferences//entry)[1]` though to avoid the wildcard.

**Comment by riedde** [comment]
This should also work (just a variation):
`$preferences//(pref:entry|entry)`

I use this when I have to deal with TEI and MEI in the same XPath especially when the element names are different.
